recipe/meta.yaml

```diff
@@ -73,9 +73,11 @@ requirements:
 test:
   commands:
     - test -f $PREFIX/lib/libfltk.a  # [unix]
+    - test -f $PREFIX/lib/libfltk${SHLIB_EXT}  # [unix]
     - test -f $PREFIX/lib/libfltk_gl.a  # [unix]
-    - if not exist %LIBRARY_LIB%\\fltk.lib exit 1  # [win]
```

Comment:
Why remove this?

Reply:
Isn't the static library file name changed here? https://github.com/conda-forge/fltk-feedstock/pull/12/files#diff-54cf74e113dd3f6d11e092fdb1d888ec82c69bdafbb15cfb6570c83ecad28f33R23-R26